app/store/index.js

```diff
@@ -68,7 +68,27 @@ export default function configureAppStore(initialState) {
             };
         },
         null,
-        {whitelist: ['views']} // Only run this filter the views state (or any other entry that ends up being named views)
+        {whitelist: ['views']} // Only run this filter on the views state (or any other entry that ends up being named views)
+    );
+
+    const emojiBlackList = {nonExistentEmoji: true};
```

Comment:
One day, we need to find a better library for this. It would be so much nicer if we could just list the parts of the store to be ignored like
```
const ignoredState = {
    'state.entities.emojis.emojiBlackList',
    ...
};
```
instead of having to construct this big function for it

Reply:
Agreed, it's super annoying to do it this way